addons/website/views/website_visitor_views.xml

```diff
@@ -128,26 +127,7 @@
                                 </span>
                                 <div t-att-class="record.page_count.raw_value ? '' : 'text-muted'">Visited Pages</div>
                             </div>
-                            <div class="w_visitor_kanban_actions_ungrouped col-lg-3 col-sm-12 py-0 my-2 text-lg-end">
-                                <button name="action_send_mail" type="object"
-                                        class="btn btn-secondary" invisible="not email">
-                                        Email
-                                </button>
-                            </div>
-                        </div>
-                        <!-- displayed in grouped mode -->
-                        <div class="o_kanban_grouped p-2">
-                            <span class="fa fa-circle text-success float-end" t-if="record.is_connected.raw_value" aria-label="Online" title="Online"/>
-                            <span class="fa fa-circle text-danger float-end" t-else="" aria-label="Offline" title="Offline"/>
-                            <div class="d-flex">
-                                <field t-if="record.country_id.raw_value" name="country_flag" widget="image_url" class="o_country_flag" alt="Country"/>
-                                <field name="display_name" class="fw-bolder"/>
-                            </div>
-                            <div class="mb-2">Active <field name="time_since_last_action"/></div>
-                            <div t-if="record.page_count.raw_value">Last Page<field name="last_visited_page_id" class="float-end fw-bold"/></div>
-                            <div>Visits<field name="visit_count" class="float-end fw-bold"/></div>
-                            <div t-if="record.page_count.raw_value" id="o_page_count">Visited Pages<span class="float-end fw-bold"><field name="page_count"/></span></div>
-                            <div name="w_visitor_kanban_actions" class="d-flex gap-1">
+                            <div id="w_visitor_kanban_actions" class="col-lg-3 col-sm-12 py-0 my-2 text-lg-end">
```

Comment:
Can't we keep a class and use the class for the xpathing?

We generally don't use IDs, less flexible, uniqueness constraint, ...

It's either a class if style + xpathing or "name" if just xpathing.